api/src/generate-service-worker.js

```diff
@@ -51,65 +50,63 @@ const getServiceWorkerHash = async () => {
 };
 
 const getLoginPageContents = async () => {
-  try {
-    return await loginController.renderLogin();
-  } catch (err) {
-    logger.error('Error rendering login page %o', err);
-    // default to returning the file
-    return [path.join(apiSrcDirectoryPath, 'templates', 'login', 'index.html')];
-  }
+  return await loginController.renderLogin();
```

Comment:
I dug down into the renderlogin function and found it was always gracefully handling errors so we don't need this handling code.